xlive/Blam/Engine/game/game_engine_default.cpp

```diff
@@ -0,0 +1,206 @@
+#include "stdafx.h"
+#include "game_engine_default.h"
+
+e_game_engine_type c_game_engine::get_type()
+{
+	return _game_engine_type_none;
+}
+
+bool c_game_engine::setup()
+{
+	return false;
+}
+
+bool c_game_engine::cleanup()
+{
+	return false;
+}
+
+bool c_game_engine::function_4()
+{
+	return true;
+}
+
+bool c_game_engine::verify_netpoint(uint32 netpoint_index)
+{
+	return true;
+}
+
+void c_game_engine::player_join(datum player_index)
+{
+
+}
+
+void c_game_engine::player_leave(datum player_index)
+{
+
+}
+
+void c_game_engine::player_rejoin(datum player_index)
+{
+
+}
+
+void c_game_engine::swap_player_indices(uint32 old_index, uint32 new_index)
+{
+
+}
+
+void c_game_engine::player_team_change(datum player_index)
+{
+
+}
+
+void c_game_engine::game_start()
+{
+
+}
+
+void c_game_engine::function_14(datum player_index)
+{
+
+}
+
+bool c_game_engine::player_can_interact_with_weapon(datum player_index, datum weapon_index)
+{
+	return true;
+}
+
+void c_game_engine::handle_player_objective_touch_interaction(datum player_index, datum object_index)
+{
+
+}
+
+void c_game_engine::update()
+{
+
+}
+
+real32 c_game_engine::get_player_speed_modifier(datum player_index)
+{
+	return 1.f;
+}
+
+uint32 c_game_engine::function_21(datum object_index)
+{
+	LOG_INFO_GAME("[{}]", __FUNCTION__);
```

Comment:
remove logging, this should probably be returning a signed int with NONE instead